Unix/configure

```diff
@@ -464,6 +464,7 @@ else
     configuuid="`echo $USER`"
 fi
 
+
```

Comment:
doesn't look that way.
if [ -x "/bin/uuidgen" -o -x "/usr/bin/uuidgen" ]; then

    if [ "$os" != "DARWIN" ]; then
        configuuid="`uuidgen`"
    else
        configuuid="`echo $USER`"
    fi

else
    configuuid="`echo $USER`"
fi